guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -61,24 +61,12 @@ The bulk of storage resides in the `/var/lib/pulp/` directory.
 These end points are not manually configurable.
 Ensure that storage is available on the `/var` file system to prevent storage problems.
 
-ifdef::satellite[]
-.Software Collections
-
-Software collections are installed in the `/opt/rh/` and `/opt/theforeman/` directories.
-
-Write and execute permissions by the root user are required for installation to the `/opt` directory.
```

Comment:
Isn't that still true, given Puppet is still installed in `/opt`?

But then again, it's the wrong "section" anyways :woman_shrugging:

Reply:
Hmm, fair point but now I wonder what to do. Should we drop it or add a new section?

Reply:
I'd drop it like a potato and see who's up to finding it again ;)